src/backend/aspen/database/models/pathogens.py

```diff
@@ -31,6 +33,11 @@ class Pathogen(idbase):  # type: ignore
     phylo_runs = relationship("PhyloRun", back_populates="pathogen")  # type: ignore
     phylo_trees = relationship("PhyloTree", back_populates="pathogen")  # type: ignore
 
+    @classmethod
+    async def get_by_slug(cls, db: AsyncSession, slug: str) -> Pathogen:
+        resp = await db.execute(sa.select(Pathogen).where(Pathogen.slug == slug))  # type: ignore
+        return resp.scalars().one()
+
```

Comment:
Helper method for getting pathogens by slug.

Reply:
why a class method and not a regular method for this?

Reply:
Because we're trying to *get* a pathogen instance from the db based on a slug. We don't *have* an instance yet.

Reply:
ah ok got it